lib/tx.js

```diff
@@ -74,10 +76,13 @@ function getPollFn(sdk, res, ref) {
     // Get the poll function
     var pollLink = util.getLink(res, 'next', 'poll');
     function pollFn() {
-      var href = pollLink.href;
+      var opts = {
+        'autoPush': autoPush
+      };
       if (rememberDevice) {
-        href += '?rememberDevice=true';
+        opts.rememberDevice = rememberDevice;
```

Comment:
Should we explicitly set this to true? Same question for autoPush and updatePhone

Reply:
I think we should, so we don't change the current way this works for ```rememberDevice```. And if we do the same for the other variables, then we will be consistent everywhere, plus they are supposed to be booleans (I am assuming that is also what the API expects)